BP/common/religion/religions/00_donyipoloism.txt

```diff
@@ -1,17 +1,17 @@
-﻿#LT-Rascek: Taken from herkles "Dharmachakra"
-#and modified; moved bathouism and sanamahism 
-#into this family as well; they are all
-#Tibeto-Burman faiths
+﻿#LT-Rascek: Taken from herkles "Dharmachakra" and modified; moved bathouism and sanamahism into this family as well; 
+# they are all Tibeto-Burman faiths
```

Comment:
Is Sanamahi really Tibeto-Burman?

Reply:
It's the ethnic religion of the Meitei people and the Meitei people are Tibeto-Burman.

Reply:
Manipur is generally not considered Burman, hence my interrogation. It is a detail nonetheless.

Reply:
It's not Burman but Meitei is a Tibeto-Burman language. Tibeto-Burman, at least the way I put it together, is the catch-all for cultures that are not properly Tibetan nor properly Burman. So the Pyu and Meitei fall into that category.

Reply:
Ok